seqr/management/commands/copy_project_tags.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db.models.query_utils import Q
+
+from seqr.models import Project, VariantTagType
+from seqr.model_utils import create_seqr_model
```

Comment:
This isn't used

Reply:
have removed the unused imports in latest commits